pkg/server/streaming.go

```diff
@@ -112,3 +130,80 @@ func handleResizing(resize <-chan remotecommand.TerminalSize, resizeFunc func(si
 		}
 	}()
 }
+
+func newTLSCert() (tls.Certificate, error) {
```

Comment:
Add comment to explain what we are doing here, and add TODO.

Reply:
done